frontend/src/components/EquipmentItemCard.tsx

```diff
@@ -0,0 +1,20 @@
+import {EquipmentItem} from "../model/EquipmentItem";
+import {Card} from "react-bootstrap";
+
+
+export type EquipmentItemCardProps = {
+    equipmentItem: EquipmentItem
+}
+
+export default function EquipmentItemCard({equipmentItem}: EquipmentItemCardProps) {
+    return <div>
```

Comment:
```suggestion
    return 
    <div>
```
If i write nothing is not good :P